tools/licenses/Apache-2.0

```diff
@@ -0,0 +1,15 @@
+# sed -E -n
+# -e 's/^(Copyright )\(c\)/\1/'
+Copyright <year> <copyright holders>
```

Comment:
Yes, this is something called "Standard License Header" (see spdx).  The `license-detector` have no direct support for this yet. Files in the `licenses` directory must match the full license text, otherwise `license-detector` won't work as expected and designed.

BTW, you could add grep for any text directly to the `license-detector` file to handle special cases like this one (please note there is already basic handling for `Apache-2.0`).  Or, the other possible option is to add support for "Standard License Header" into the `license-detector`.  For example you could create directory `license-headers` with such headers, or put headers into the `licenses` directory, but name such headers differently to not clash with full-text licenses (for example `header-Apache-2.0`).  I think the first option (`license-headers` directory) is better.

Reply:
I added the license-headers directory, but I think we should clarify somehow this is meant for the "Standard License Header" files. Or maybe it's clear enough like this.

I also misunderstood how the sed worked, fixed that too.

Reply:
> I added the license-headers directory, but I think we should clarify somehow this is meant for the "Standard License Header" files. Or maybe it's clear enough like this.

Feel free to propose how to clarify that.  One possible option is to add a comment to the `license-detector` script, but I'm open for other solutions too.

BTW, I think it would be good if the commit message says you add header only, not the full license text.